app/ui/browser-modern/setup-ipc.js

```diff
@@ -47,19 +48,25 @@ export default function({ store, userAgentClient }) {
   });
 
   ipcRenderer.on('go-back', () => {
-    if (!UISelectors.getOverviewVisible(store.getState())) {
+    const pageId = PagesSelectors.getSelectedPageId(store.getState());
+    const pageState = PagesSelectors.getPageState(store.getState(), pageId);
```

Comment:
Yeah to be really strict about that it seems we'd need selectors for getPageCanGoBack, getPageCanRefresh, etc (which seems like overkill), as we eventually end up directly working with the state whether it's accessing `page.state.canGoBack` or `getPageState().canGoBack`. I'd like to hear @linclark's thoughts on how she'd handle this case.